.github/workflows/coverity.yml

```diff
@@ -0,0 +1,89 @@
+name: Coverity
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+permissions: read-all
+
+jobs:
+  coverity:
+    name: Coverity
+    runs-on: [Linux, build]
+    container:
+      image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:alldeps
+      options: -u 1001:1001
+    env:
+      CCACHE_DIR: ${{ inputs.build_cache_root }}/build_cache_${{ inputs.build_cache_suffix }}
+      CCACHE_MAXSIZE: 8G
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        sparse-checkout: |
+          devops/actions
+
+    - name: Register cleanup after job is finished
+      uses: ./devops/actions/cleanup
+
+    - uses: ./devops/actions/cached_checkout
+      with:
+        path: src
+        ref: ${{ github.sha }}
+        cache_path: "/__w/repo_cache/"
+
+    - name: Get coverity tool
+      run: |
+        wget https://scan.coverity.com/download/linux64 --post-data "token=${{ secrets.COVERITY_TOKEN }}&project=intel%2Fllvm" -O coverity_tool.tgz
+        tar -xf coverity_tool.tgz
```

Comment:
It doesn't require a separate cleanup right?

Reply:
`wget <args> -O - | tar -xzvf -`